Artiom/src/main/java/Circle_Area.java

```diff
@@ -0,0 +1,8 @@
+public class Circle_Area {
```

Comment:
Don't use underscore in class's name
Naming convention: Class names should be nouns with the first letter of each internal word capitalized.

Reply:
Done